slack_bolt/logger/messages.py

```diff
@@ -186,33 +197,14 @@ def warning_unhandled_request(  # type: ignore
             or filtered_body.get("view", {}).get("callback_id")  # type: ignore
             or "your-callback-id"
         )
-        return _build_unhandled_request_suggestion(
-            default_message,
-            f"""
```

Comment:
The sample code snippets felt like they were cluttering the function, I moved them to a separate file